doc/users/next_whats_new/collection_offsets.rst

```diff
@@ -0,0 +1,5 @@
+Setting collection offset transform after initialization
+--------------------------------------------------------
+`~matplotlib.collections.Collection.set_offset_transform` was added.
+
+Previously the offset transform could not be set after initialization.
```

Comment:
It's a rare use case, so I was hesitant to include it, but when creating a collection outside an axes, you might want to set `transOffset` to `ax.transData`. I've added a mention of it now. There could be other use cases, since it's a generic function.